### PR TITLE
Added Translation for maintainer's guide in Russian

### PR DESCRIPTION
**Pull request by Yash-sudo-web**

### CODE REVIEW

File: translations/maintainer_guide/maintainer_guide.rus.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Руководство сопровождающего
+
+Это руководство предназначено для тех, кто хотел бы присоединиться к проекту в качестве сопровождающих, чтобы помочь сообществу управлять проектом и поддерживать его. (Это руководство не для **начинающих участников**.)
+
+## Translations
+
+Это руководство также доступно на [других языках](translations/README.md)
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.rus.md:7]:
```suggestion
Это руководство также доступно на [других языках](/translations/README.md)
```

File: translations/maintainer_guide/maintainer_guide.rus.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+| [Английский](maintainer_guide.md)   | [Бангла](translations/maintainer_guide/maintainer_guide.ben.md) | [Китайский (традиционный)](/translations/maintainer_guide/maintainer_guide.zho-tc.md) | [Немецкий](translations/maintainer_guide/maintainer_guide.ger.md) | [хинди](translations/maintainer_guide/maintainer_guide.hin.md) |
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.rus.md:9]:
```suggestion
| [Английский](/maintainer_guide.md)   | [Бангла](/translations/maintainer_guide/maintainer_guide.ben.md) | [Китайский (традиционный)](/translations/maintainer_guide/maintainer_guide.zho-tc.md) | [Немецкий](/translations/maintainer_guide/maintainer_guide.ger.md) | [хинди](/translations/maintainer_guide/maintainer_guide.hin.md) |
```

File: translations/maintainer_guide/maintainer_guide.rus.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+| :---: | :---: | :---: | :---: | :---: |
+| [итальянский](translations/maintainer_guide/maintainer_guide.ita.md) | [Японский](translations/maintainer_guide/maintainer_guide.jpn.md) | [португальский](translations/maintainer_guide/maintainer_guide.por.md) | [украинец](/translations/maintainer_guide/maintainer_guide.ukr.md) |
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.rus.md:11]:
```suggestion
| [итальянский](/translations/maintainer_guide/maintainer_guide.ita.md) | [Японский](/translations/maintainer_guide/maintainer_guide.jpn.md) | [португальский](/translations/maintainer_guide/maintainer_guide.por.md) | [украинец](/translations/maintainer_guide/maintainer_guide.ukr.md) |
```

File: translations/maintainer_guide/maintainer_guide.rus.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+> Приветствуются переводы проектной документации. Прочтите [`Руководство по переводу`](translations/README.md) внести свой вклад.
```
Comment (TenTraicion) [inline translations/maintainer_guide/maintainer_guide.rus.md:13]:
```suggestion
> Приветствуются переводы проектной документации. Прочтите [`Руководство по переводу`](/translations/README.md) внести свой вклад.
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
the links should be fixed before merging

**Comment by Yash-sudo-web** [comment]
heyy @TenTraicion, requested changes have been done

**Review by TenTraicion — APPROVED** [review]

**Pull request merged**